webpack-env missing path

I'm using the latest typescript version and due to the addition from **path** to node in 12.12.48 I get this error in webpack-env:

```
node_modules/@types/webpack-env/index.d.ts:349:15 - error TS2430: Interface 'NodeJS.Module' incorrectly extends interface '__WebpackModuleApi.Module'.
  Types of property 'parent' are incompatible.
    Type 'Module | null' is not assignable to type 'NodeModule | null'.
      Property 'path' is missing in type 'Module' but required in type 'NodeModule'.

349     interface Module extends __WebpackModuleApi.Module {}
                  ~~~~~~

  node_modules/@types/node/globals.d.ts:236:5
    236     path: string;
            ~~~~
    'path' is declared here.
```

@use-strict @rhonsby

--- a/types/node/v12/globals.d.ts
+++ b/types/node/v12/globals.d.ts
@@ -1162,6 +1162,12 @@ declare namespace NodeJS {
         loaded: boolean;
         parent: Module | null;
         children: Module[];
+        /**
+         * @since 11.14.0
+         *
+         * The directory name of the module. This is usually the same as the path.dirname() of the module.id.
+         */
+        path: string;
         paths: string[];
 
         constructor(id: string, parent?: Module);

--- a/types/node/v12/node-tests.ts
+++ b/types/node/v12/node-tests.ts
@@ -764,7 +764,7 @@ import moduleModule = require('module');
     const m2: Module = new Module.Module("moduleId");
     const b: string[] = Module.builtinModules;
     let paths: string[] = module.paths;
-    const path = module.path;
+    const path: string = module.path;
     paths = m1.paths;
 
     const customRequire1 = moduleModule.createRequireFromPath('./test');

--- a/types/node/v12/node-tests.ts
+++ b/types/node/v12/node-tests.ts
@@ -764,6 +764,7 @@ import moduleModule = require('module');
     const m2: Module = new Module.Module("moduleId");
     const b: string[] = Module.builtinModules;
     let paths: string[] = module.paths;
+    const path = module.path;
     paths = m1.paths;
 
     const customRequire1 = moduleModule.createRequireFromPath('./test');

